tap_facebook/__init__.py

```diff
@@ -284,6 +285,7 @@ class Ads(IncrementalStream):
     '''
 
     key_properties = ['id', 'updated_time']
+    replication_method = 'INCREMENTAL'
```

Comment:
Is it possible to define it in the `IncrementalStream` class instead of defining it in all classes?

Reply:
Defined incremental replication key in  `IncrementalStream` class and removed from extended class. Leads and AdsInsights are incremental but extended from `Stream` so keep it as it is.